core/src/queue.rs

```diff
@@ -444,14 +450,16 @@ mod tests {
     }
 
     #[test]
```

Comment:
You didn't use `tokio::test` is that intentional?

Reply:
I use `tokio::test` in places where `LiveQueryStore` is needed, because it spawns a new task. And this test doesn't need `LiveQueryStore`

Reply:
I imported `tokio::test`. If you ask why sometimes I don't import it, then it's because not all tests need `LiveQueryStore` which requires `tokio` runtime. But this test requires it and provided `#[test]` attribute fallbacks to `#[tokio::test]`

Reply:
I missed your import.